third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -908,6 +908,17 @@ func resourceContainerCluster() *schema.Resource {
 				},
 			},
 
+<% unless version == 'ga' -%>
+			"networking_mode": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice([]string{"VPC_NATIVE", "ROUTES"}, false),
+				Default:      "ROUTES",
```

Comment:
Instead of defaulting to `ROUTES`, I think we need to make this O+C. Users with VPC-native clusters today will see a dangerous diff, otherwise! The empty string will evaluate as false for both `UseIpAliases` and `UseRoutes`, which will correspond to letting the API choose.

Reply:
Oh, yes, good call!  Thanks!